doc/sphinx/source/data/new-commondata.rst

```diff
@@ -0,0 +1,155 @@
+Organization following the naming convention
```

Comment:
Is the convention enforced mechanically? Is it needed to process the names?

Reply:
> Is it needed to process the names?

Yes. The folder in which the data is will always be `name.rsplit("_", 1)[0]` and the choice of the observable `name.rsplit("_", 1)[1]